elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/ControllerTest.java

```diff
@@ -463,4 +466,13 @@ public void graphqlTestForbiddenCreate() {
                 .body("errors.message", contains("CreatePermission Denied"))
                 .statusCode(200);
     }
+
+    // Controller disabled by default.
+    @Test
+    public void exportControllerTest() {
+        when()
+                .get("/export/asyncQueryId")
```

Comment:
I'm not sure this is testing the controller is disabled or just that asyncQueryId doesn't exist as an export object.

Reply:
This is for disabled controller

Reply:
They both send 404 so not sure how to separate it out. In case of controller disabled we will get an `error` message `Not Found`. Does not happen in case of query not found.

Reply:
What happens if you post an export request?